solr/solr-ref-guide/src/shard-management.adoc

```diff
@@ -21,10 +21,66 @@ In SolrCloud, a shard is a logical partition of a collection. This partition sto
 
 The number of shards you have helps to determine how many documents a single collection can contain in total, and also impacts search performance.
 
+The examples assume you have started Solr via `bin/solr start -c -e techproducts`.
+
 [[splitshard]]
 == SPLITSHARD: Split a Shard
 
-`/admin/collections?action=SPLITSHARD&collection=_name_&shard=_shardID_`
+[.dynamic-tabs]
+--
+
+[example.tab-pane#v1splitshard]
+====
+[.tab-label]*V1 API*
+
+*Input*
+
+[source,text]
+----
+http://localhost:8983/solr/admin/collections?action=SPLITSHARD&collection=techproducts&shard=shard1
+----
+
+*Output*
+
+[source,xml]
+----
+<response>
```

Comment:
[0] ditto, re: might be worth either changing the output to JSON or adding `wt=xml` to the request